### ISSUE TITLE
Bug:  Looks bad on tab

### ISSUE THREAD

**Issue by Quantum-Codes** [Scratch-Bookmarklets/Scratch-Bookmarklets.github.io#199]
**Describe the bug**
Just looks bad on tab/mobile

Too much crowded
![Screenshot_20210727-170857_Chrome](https://user-images.githubusercontent.com/87054411/127147674-5ae89776-f0be-456f-82f8-1480b4c1418d.jpg)
![Screenshot_20210727-170846_Chrome](https://user-images.githubusercontent.com/87054411/127147692-28e38a9b-e606-4fef-8322-57fd815068b0.jpg)

**To Reproduce**
Go to the website on a tab or mobile

1. Grab your handheld device
2. open a browser
3. go to the website
4. done
**Expected behavior**
Look good or decent???????????

Agh
**Browser Information**
Chrome.

**Additional context**
No..

**Comment by xItzDiamondx** [comment]
I dont know. I tested it on my phone, and, yeah it looks a bit crowded. We can fix that with some simple JS that detects the device width.![Screenshot_20210728-224534_Samsung Internet.jpg](https://user-images.githubusercontent.com/81493936/127428391-f1a952b3-e23e-4554-88b8-de7e5a32caba.jpg)

**Comment by Quantum-Codes** [comment]
Ok

**Comment by NotDiamondZ** [comment]
I believe it is best if we create a diverse repository for the mobile users since the JS is far too intricate for me to compute. I also believe that we should create a separate repository for the bookmarks. This repository holds the website, not the bookmarklets.

**Comment by Shluffy** [comment]
> I believe it is best if we create a diverse repository for the mobile users since the JS is far too intricate for me to compute. I also believe that we should create a separate repository for the bookmarks. This repository holds the website, not the bookmarklets.

will work on that

**Comment by NotDiamondZ** [comment]
Been 6 days. How's it going? It's best if you add me as a org member pls

**Comment by Quantum-Codes** [comment]
Done?

**Comment by Shluffy** [comment]
No, sorry. I have been very busy, and am not experienced with mobile things. I don't think this is really meant for mobile usage, although it still looks bad. I will work on a pop-up that recommends that you view this on a computer

**Comment by Quantum-Codes** [comment]
Ok
